server/auth/types/jwt/jwt_auth.ts

```diff
@@ -87,9 +85,9 @@ export class JwtAuthentication extends AuthenticationType {
     return false;
   }
 
-  protected getAdditionalAuthHeader(
+  async getAdditionalAuthHeader(
```

Comment:
why this needs to be async?

Reply:
multi auth requires to get cookie from` sessionStorageFactory`. So I defined `getAdditionalAuthHeader` as async abstract method in authentication_type.ts